CRM/Civicaseextras/AngularModifiers/CaseDuration.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use \Civi\Angular\Manager as AngularManager;
+use \Civi\Angular\ChangeSet as AngularChangeSet;
+
+class CRM_Civicaseextras_AngularModifiers_CaseDuration {
+  protected $angular;
+  protected $caseDuration;
+
+  /**
+   * @param AngularManager $angular as provided by the alter angular hook.
+   */
+  public function __construct(AngularManager &$angular) {
+    $this->angular = $angular;
+  }
+
+  /**
+   * Adds the case duration column and value to the case list table template of civicase.
+   */
+  public function runModifications() {
+    $this->caseDuration = _civicaseextras_get_caseDurationField();
```

Comment:
I'm not sure case duration should be a property here. If someone else adds another public method without this call it would not be set. You could just pass it into the methods here instead of setting it as a property. Also, as mentioned above, can you avoid using function calls here and either move `_civicaseextras_get_caseDurationField` into a private method or separate class please?